bin/minifollowups/pycbc_foreground_minifollowup

```diff
@@ -178,7 +184,7 @@ while event_count < num_events and curr_idx < (events_to_read - 1):
                         tags=['SKIP_{}'.format(event_count)]),)
         skipped_data = []
 
-    bank_id = f['{}/template_id'.format(file_val)][:][sorting][curr_idx]
+    bank_id = bank_ids[curr_idx]
```

Comment:
Removing the `[sorting]` indexing is intentional right?

Reply:
Yeah that's right, bank_ids is already sorted by `event_idx` above